src/cljs/main/broadfcui/common/table/table.cljs

```diff
@@ -86,10 +88,11 @@
    :render
    (fn [{:keys [props state]}]
      (let [props (utils/deep-merge default-props props)
-           {:keys [rows column-display total-count filtered-count query-params loading?]} @state
-           {:keys [body toolbar paginator]} props
+           {:keys [rows column-display filtered-count query-params loading?]} @state
+           {:keys [body toolbar paginator total-count]} props
            {:keys [empty-message columns behavior external-query-params]} body
            {:keys [fixed-column-count allow-no-sort?]} behavior
+           total-count (or total-count (:total-count @state))
```

Comment:
Another option:
```clojure
total-count (some :total-count [props @state])
```
for slightly more consistency instead of destructuring it from props and not state.